dependencies/focal/ansible-runner/changelog

```diff
@@ -1,12 +1,65 @@
-ansible-runner (1.4.7-1~tfm+1) sid; urgency=medium
+ansible-runner (2.2.1-1) stable; urgency=low
```

Comment:
```suggestion
ansible-runner (2.2.1-1~tfm+1) stable; urgency=low
```

Given we're fiddling around with an official package, I'd prefer to keep the `~tfm+N` suffix in the version, especially as it will allow users to upgrade to the official package once that is present.